djstripe/templates/djstripe/account.html

```diff
@@ -27,15 +27,6 @@
 
 
 
```

Comment:
I'm confused about all of the changes to this file in this revision. What's your reasoning behind them?